discovery-provider/integration_tests/notifications/test_repost.py

```diff
@@ -27,14 +27,14 @@ def test_repost_notification(app):
                 "user_id": 1,
                 "repost_item_id": 100,
                 "repost_type": RepostType.track,
+                "is_repost_repost": False,
```

Comment:
don't think this is necessary since you adjusted populate_mock_db. not your change we're also doing populate_mock_db twice lol